### PR TITLE
split tests for data module

### PR DESCRIPTION
**Pull request by ahartikainen**
Simplify tests.

This PR enables one to test specific modules without other ones.

### CODE REVIEW

**Comment by canyon289** [comment]
The direction of this LGTM

**Comment by ahartikainen** [comment]
Ready for review

File: arviz/tests/test_plots.py [hunk 1]
```diff
@@ -34,14 +33,57 @@
...
 @pytest.fixture(scope="module")
-def models(eight_schools_params):
+def models():
     class Models:
-        models = load_cached_models(eight_schools_params, draws=500, chains=2)
-        pymc3_model, pymc3_fit = models["pymc3"]
-        stan_model, stan_fit = models["pystan"]
-        emcee_fit = models["emcee"]
-        pyro_fit = models["pyro"]
+        model_1 = create_model(seed=10)
```
Comment (canyon289) [inline arviz/tests/test_plots.py]: Why two models?

**Comment by canyon289** [comment]
This PR is amazing

File: arviz/tests/test_plots.py (hunk 1 above)
Comment (ahartikainen) [reply arviz/tests/test_plots.py]: Some plots test multiple models, and this way, if dev saves the plot, the models won't overlap.

**Comment by canyon289** [comment]
LGTM. Im fine with merging

**Pull request merged**